app/src/api.ts

```diff
@@ -174,12 +174,32 @@ export interface NewChart {
   Five: number;
 }
 
-//for Popular vendor in neighborhood by rating
+//Graph 5: Top 10 Vendors in a certain Area
 export interface PopularVendor {
   TotalRatings: number;
   BusinessName: string;
   Location: string;
 }
+//Graph 4: Top 3 popular Cuisine Types by Area
+export interface PopularCuisine {
+  CuisineType: string;
+  TotalRating: number;
+  Location: string;
+}
+//Graph 3: Top 5 Popular Searching Queries in a certain month
+export interface PopularSearch {
+  QueryText: string;
+  Month: DateTime;
+  TotalSearch: number;
+}
+
+//Graph 1: Timeline of Average Rating Increase or Decrease
+export interface AverageRating {
```

Comment:
The type of `AverageRating` should be string to match the `AverageRatingByMonth` definition.

Reply:
Do you mean the Month should be string because I converted it to the actual word? I don't thing AverageRating should be string because it's the actual average calc so it should be number, no?

Reply:
Sorry, `Month` should be a string to match the backend definition.

Reply:
lol, ok pushing it through now